hellosign_sdk/hsclient.py

```diff
@@ -674,6 +681,13 @@ def send_signature_request_embedded(self, test_mode=False, client_id=None,
 
             signing_options (dict, optional): Allows the requester to specify the types allowed for creating a signature. Defaults to account settings.
 
+            attachments (list of dict):            A list of attachments, which each has the following attributes:
```

Comment:
Can you update this block with the suggestions from `send_signature_request`?

Reply:
@oconnor-sn can you address this comment too please. It should be
`A list of attachments, each with the following attributes:`

Reply:
The version I have with the latest accepted suggestions show that this has been updated.
I reviewed these changes and all occurrences match against your suggestions.